<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BBluetooth</class>
 <widget class="QDialog" name="BBluetooth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="toggle_bluetooth">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QLabel" name="bluetooth_icon">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>251</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="bluetooth_state">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_Home">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>101</width>
     <height>661</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/icons/home.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
